react/wonderland/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "wonderland-react-example",
+  "version": "1.0.0",
+  "description": "My Wonderland project",
+  "main": "js/index.js",
+  "type": "module",
+  "module": "js/index.js",
+  "scripts": {},
+  "keywords": [
+    "wonderland-engine"
+  ],
+  "dependencies": {
+    "@wonderlandengine/api": "^1.1.5",
```

Comment:
I think that should probably be a peer-dependency, since the owning package might use the API as well.